README.md

```diff
@@ -191,6 +192,18 @@ feed:
         - updates
 ```
 
+If your collections have `title` and `description` configurations variables, this plugin can consume those as well:
+
+```yml
+feed:
+  collections:
+    changes:
+      title: "Important News and Updates"
+      description: "Subscribe to this feed for exciting new changelogs."
+```
```

Comment:
This documentation doesn't reflect the implementation.
The implementation and its tests use the *metadata* defined for collection under top-level key `"collections"`, not a key nested under `"feed"`.

Reply:
Thank you for catching this error!